.size-limit.js

```diff
@@ -30,6 +30,12 @@ module.exports = [
     path: 'packages/material-ui-styles/build/index.js',
     limit: '14.9 KB',
   },
+  {
+    name: 'The size of the @material-ui/core/Popper component',
+    webpack: true,
+    path: 'packages/material-ui/build/Popper/index.js',
+    limit: '16.0 KB',
```

Comment:
The `withTheme` import for RTL support costs us 7 kB gzipped. Maybe we should look in the body dir attribute instead? cc @eps1lon?